src/apps/answers/router.py

```diff
@@ -326,3 +328,13 @@
         **AUTHENTICATION_ERROR_RESPONSES,
     },
 )(answers_existence_check)
+
+router.get(
+    "/applet/{applet_id}/validate-takenow",
```

Comment:
Can we consider the naming of this route and related class names?

Currently, the only flow we have to initiate multi-informant assessments is via the Take Now flow. It was called "Take Now" because it's triggered in the admin panel (ie. outside of where assessments are typically completed) and outside of any kind of schedule.

In the future, multiinformant assessments might initiate inside the web or mobile app directly because it was part of a schedule or there is an relationship between the respondent and the target and there's no need for an admin to setup that up via the Take Now in admin panel.

So the validation endpoint is applicable to Take Now, but also all these other multi-informant scenarios that could benefit from it too.

Reply:
Thanks so much!